docs/guides/admin/docs/workflowoperationhandlers/speech-to-text-woh.md

```diff
@@ -13,13 +13,14 @@ only [Vosk](https://alphacephei.com/vosk/) available as STT Engine. The subtitle
 Parameter Table
 ---------------
 
-|configuration keys|required|description                                                                      |
-|------------------|--------|---------------------------------------------------------------------------------|
-|source-flavor     |yes     |The source media package to use                                                  |
-|target-flavor     |yes     |Flavor of the produced subtitle file                                             |
-|target-element    |no      |Define where to append the subtitles file. Possibilities are: as a 'track' or as an 'attachment'. The default is "attachment".|
-|language-code     |no      |The language of the video or audio source (default is "eng). It has to match the name of the language model directory. See 'vosk-cli'.|
-|target-tags       |no      |Tags for the subtitle file                                                       |
+|configuration keys|required| description                                                                                                                             |
+|------------------|--------|-----------------------------------------------------------------------------------------------------------------------------------------|
+|source-flavor     |yes     | The source media package to use                                                                                                         |
+|target-flavor     |yes     | Flavor of the produced subtitle file. The subflavor supports the language-code placeholder `#{lang}`                                    |
+|target-element    |no      | Define where to append the subtitles file. Possibilities are: as a 'track' or as an 'attachment'. The default is "attachment".          |
+|language-code     |no      | The language of the video or audio source (default is "eng"). It has to match the name of the language model directory. See 'vosk-cli'. |
```

Comment:
This pull request changes the default, doesn't it? It's no longer `eng`, but whatever is in the DC catalog. This documentation isn't quite correct, is it? Therefore, this pull request should also go into `develop`

Reply:
It does not change the default behavior. The default fallback is still eng; it’s only configurable now. It was using the dc catalog before if the lang code was not defined.

Reply:
The behavior is just not documented. See function getMediaPackageLanguage, and I just changed line 287.

https://github.com/geichelberger/opencast/blob/2bf10e68635020ba4f0ee07c7bc094dbb4f6329a/modules/speech-to-text-workflowoperation/src/main/java/org/opencastproject/workflow/handler/speechtotext/SpeechToTextWorkflowOperationHandler.java#L268-L291